rotkehlchen/tests/unit/decoders/test_scroll_bridge.py

```diff
@@ -0,0 +1,474 @@
+            location_label=user_address,
+            notes='Burned 0.000383984852231895 ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=25,
+            timestamp=TimestampMS(1708534727000),
+            location=Location.SCROLL,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset('eip155:534352/erc20:0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4'),
+            balance=Balance(amount=FVal('450')),
+            location_label=user_address,
+            notes='Bridge 450 USDC from Scroll to Ethereum via Scroll bridge',
+            counterparty=CPT_SCROLL,
+            address=L2_USDC_GATEWAY,
+        ),
+    ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x59ceD3eeB1cb01AfE92ADbC994D1CE78310E668E']])
+def test_receive_erc20_on_ethereum(database, ethereum_inquirer, ethereum_accounts):
+    evmhash = deserialize_evm_tx_hash('0xbe6e98db62ee719922c7d01b0b623e16c8ee162a2bf9143603f11eba3e3dd474')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    user_address = ethereum_accounts[0]
+    assert events == [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=TimestampMS(1708223951000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.002531473518821568')),
+            location_label=user_address,
+            notes='Burned 0.002531473518821568 ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=549,
+            timestamp=TimestampMS(1708223951000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset('eip155:1/erc20:0xdAC17F958D2ee523a2206206994597C13D831ec7'),
+            balance=Balance(amount=FVal('1647')),
+            location_label=user_address,
+            notes='Bridge 1647 USDT from Scroll to Ethereum via Scroll bridge',
+            counterparty=CPT_SCROLL,
+            address=L1_ERC20_GATEWAY,
+        ),
+    ]
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xd32dEe97C7D7Fdfb826CC3bB210dD009D2750ae2']])
+def test_deposit_send_message_ethereum(database, ethereum_inquirer, ethereum_accounts):
+    evmhash = deserialize_evm_tx_hash('0x7ae1e3ef6e4d87e2e44446a683b9ca624241a24d294b60ddec39af1309aeb37a')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    user_address = ethereum_accounts[0]
```

Comment:
For all the tests in this file it would be nice and would save space if you took common attributes out as variables. Such as timestamp, gas, amount etc. And reuse it in the events below.

Can see it in many other decoder tests such as this one: https://github.com/rotki/rotki/blob/341b2ed4c1926190b40f5c748d7074192eab76c9/rotkehlchen/tests/unit/decoders/test_gnosis_sdai.py#L32